doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,137 @@ Topics include:
+* Individual Virtual Machines and Containers
+* Virtualization infrastructure components
+* Physical servers (compute) and network elements
+* Toolservers with their applications (DNS, Identity Management, Zero Touch Provisioning, etc) 
+* Cabling 
+
+<a name="9.6.1.3"></a>
+#### 9.6.1.3. Data categories
+
+There are three main observability categories: metrics, alarms and logs:
+
+1. **Metrics** or telemetry report counters and gauge levels and can either be pulled periodically e.g. via SNMP or REST, or pushed as streams using gRPC, NETCONF, which receivers registered for certain sensors, or by registering as a publisher to a Kafka bus. These messages must be structured in order to get parsed successfully.
+
+2. **Alarms** indicate state changes, categorized by severity, often with a description of what just
+hap- pened. Most common transport protocol is SNMP (traps and inform messages). They
+are generated by physical network elements as well as logical devices. In addition, they can
+also be generated by monitoring applications in form of configured thresholds or dynamic by
+means of Machine Learning (ML) algorithms - generally, they are describing anomalies.
+
+3. **Logs** messages are generated by most physical devices (compute and network) and virtual
+applications and transported over SYSLOG and tend to come in high volumes.
+
+4. **Traces** are end-to-end signalling messages (events) created to fullfil execution of requests on
+the distributed system services. OTHER WORDS: Traces are all action points executed in
+order to provide response to the request set to the distributed system service. Even the call
+can be thought of as a request which starts by INVITE message of the SIP protocol.
+
+<a name="9.6.1.4"></a>
+#### 9.6.1.4. Where to send the data
+
+If the sources of observability data have to send their data towards particular specific destinations, then it is a concept with high dependency and extremely prone to errors, especially in case of configuration changes. In order to keep it clean and elegant, data sources should be completely independent of the data receivers; i.e. data sources must not be impacted with any change in the data receivers.
+
+Ideally, observability data would be sent from the source systems always to the same destination. This is achieved by using message brokers to decouple data producers from data consumers, and that inherently means that sources send data always to the same point. It is worth noting that message brokers can be considered as a common data bus.
+
+<a name="9.6.1.5"></a>
+#### 9.6.1.5. Which protocol, interface, and format to use
+
+While protocols and interfaces are dictated by the selection of the message broker (common data bus) system, data format is usually customizable according to the needs of users. The concept of Schema Registry mechanism, well known in the world of big data, is helpful here to make sure that message structures and formats are consistently used.
```

Comment:
Can't a broker support multiple interfaces and input and output data formats including data conversion from one format to another?

Reply:
The idea was to say that if we use Kafka, then we cannot use i.e. syslog or snmp as a protocol. Neither we can use interfaces used between syslog or snmp. If Kafka is used, then we have to use interface of Kafka enpoint as well as the protocol for sending Kafka messages.

Reply:
Generally a data conversion component will be needed (e.g through a side car)